kibana-reports/common/index.ts

```diff
@@ -10,3 +10,14 @@ export const REPORTS_SCHEDULER_API = {
 export const NOTIFICATION_API = {
   SEND: '/_opendistro/_notifications/send',
 };
+
+const BASE_REPORTS_URI = '/_opendistro/_reports';
+
+export const ES_REPORTS_API = {
+  ON_DEMAND_REPORT: `${BASE_REPORTS_URI}/on-demand`,
```

Comment:
np: change `on-demand` to `on_demand` to maintain consistency

Reply:
Agree, but this will require ES-reporting plugin change, could you confirm @akbhatta ?

Reply:
sure. I will do the change.

Reply:
Updated reports-scheduler, and addressed comment in the latest commit